csuntechlab/iSTART
# STR-75 -- FE: Check authentication for Vue routes

# Description
Added navigation guarding for protected routes.
  
# Testing
1. Make sure you can ***`NOT`*** visit **`( /logout, /schedule, /admin )`** routes without authentication.
2. Make sure you ***`CAN`*** visit **`Non-existant`** routes such as ***`/blahblah`*** which will direct you to the 404 page.
3. Ensure when login in as a normal user such as _cyndee.bellamy,_ you are not able to access the **`/admin`** route but are able to access all user views.
4. Log out using the **`/logout`** route.
5. Log in using an admin account such as _steve_ , ensure you are only able to access the **`/admin`** and **`/logout`** route.
